src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -63,10 +58,13 @@ public function supportsNormalization(mixed $data, string $format = null /* , ar
      * returned as keys, and each type should be mapped to a boolean indicating
      * if the result of supportsNormalization() can be cached or not
      * (a result cannot be cached when it depends on the context or on the data.)
+     * A null value means that the normalizer does not support the corresponding
+     * type.
      *
-     * The special type '*' can be used to indicate that the normalizer might
-     * support any types. A null value means that the normalizer does not support
-     * the corresponding type.
+     * There are 3 special types:
+     *  - "object" indicates that the normalizer might support any classes;
+     *  - "native-*" indicates that the normalizer might support any native types;
```

Comment:
this documentation misses the explanation of the `native-` prefix for non-object types (`native-bool`, `native-int` and so one), which is what this `native-*` covers.

This concept of `native types` is not that clear IMO given that this is not a naming used by PHP itself (they talk about "builtin types")

Reply:
Good call. Your comment made me realize that we don't need the `native-*` wildcard; "object" is enough.